tiledb/sm/stats/stats.cc

```diff
@@ -461,6 +459,15 @@ std::string Stats::dump_read() const {
   auto read_cell_num = counter_stats_.find(CounterType::READ_CELL_NUM)->second;
   auto read_ops_num = counter_stats_.find(CounterType::READ_OPS_NUM)->second;
 
+  // Derived counters.
+  auto read_dim_num = read_dim_fixed_num + read_dim_fixed_num + read_dim_zipped_num;
```

Comment:
I see `read_dim_fixed_num` get added twice here, is that intentional?

Reply:
Did you mean to add `read_dim_var_num` instead?

Reply:
Yeah, that was what I meant to use here. Good catch!